node/cn/handler.go

```diff
@@ -30,13 +30,16 @@ import (
 	"math/big"
 	"math/rand"
 	"runtime/debug"
+	"sort"
 	"sync"
 	"sync/atomic"
 	"time"
 
 	"github.com/kaiachain/kaia/blockchain"
+	kaia_blockchain "github.com/kaiachain/kaia/blockchain"
```

Comment:
It's already imported above.

Reply:
fix at https://github.com/kaiachain/kaia/pull/681/commits/8a1df271f5d633edd14e04bc5f51ae2e00e7f7c2